spec.md

```diff
@@ -1,8 +1,8 @@
 # Open Container Initiative
```

Comment:
rename spec.md to guidance or reference

Reply:
Renamed to `artifact-authors.md` to reflect the first of three PRs for the 3 primary goals.
Subsequent PRs will address `registry-operators.md` for what it takes to support artifacts and `publishing-well-known-types.md` for the process, IANA references and examples.